### PR TITLE
Made Changes on the UX and UI

### PR DESCRIPTION
**Pull request by ryanesmores07**
Artist page is now dynamic, let me know what you think

### CODE REVIEW

File: client/src/components/SearchSection.tsx [hunk 1]
```diff
@@ -1,37 +1,67 @@
...
+//         Accept: "application/json",
+//         Authorization: `Discogs key=${key}, secret=${secret}`,
+//       },
+//     });
+//     const data = await response.json();
+//     console.log("Full Data:", data);
+//   } catch (error) {
+//     console.error("Error fetching data:", error);
+//   }
+// };
```
Comment (PavelKhomyakov) [inline client/src/components/SearchSection.tsx]: This is previous code? Why did you decided to comment it? If we have new component, we don't need it anymore.

File: client/src/components/QueryResults.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                      {title}
+                    </h2>
+                  </div>
+                </CardContent>
+              </Card>
+            </Link>
+          );
+        })}
+      </div> */}
+      {/* <PaginationContainer /> */}
```
Comment (PavelKhomyakov) [inline client/src/components/QueryResults.tsx]: Why that block also commented?

File: client/src/components/SearchSection.tsx (hunk 1 above)
Comment (ryanesmores07) [reply client/src/components/SearchSection.tsx]: Ah yea forgot to clean up, will make another pull request and delete old codes

File: client/src/components/QueryResults.tsx (hunk 2 above)
Comment (ryanesmores07) [reply client/src/components/QueryResults.tsx]: This one also will be deleted

**Pull request closed without merging**